StateInformation.c

```diff
@@ -0,0 +1,59 @@
+#include "StateInformation.h"

```

Comment:
This file contains all interface commands expected to be sent via bluetooth to the app. Need to test if nullchar works correctly